README.md

```diff
@@ -375,7 +375,7 @@ In `.husky/pre-commit`
 npx lint-staged
 ```
 
-_Note we don’t pass a path as an argument for the runners. This is important since lint-staged will do this for you._
+_Note: we don't pass a path as an argument for the runners. This is important since lint-staged will do this for you._
```

Comment:
That’s technically incorrect since it’s not the apostrophe but let’s keep it for consistency and let GitHub figure out proper typography rendering.